### PR TITLE
Add time to log, improved cores, disable html body log

### PR DESCRIPTION
**Pull request by matllubos**

### CODE REVIEW

File: security/utils.py [hunk 1]
```diff
@@ -312,31 +313,50 @@ def remove_nul_from_string(value):
...
             for cursor_block in line.split('\r'):
                 if not cursor_first:
                     self.seek(self._last_newline)
                     self.truncate()
+                    self._write_time = True
                 cursor_first = False
+
+                if self._write_time and not is_atty_string(cursor_block):
+                    cursor_block = '{}{}[{}]{} {}'.format(
+                        self.RESET, self.BOLD, now().strftime('%d-%m-%Y %H:%M:%S'), self.RESET, cursor_block
```
Comment (rubickcz) [inline security/utils.py:349]: Maybe use https://pypi.org/project/termcolor/?

Comment (matllubos) [reply security/utils.py:349]: I didn't want to use new extra library instead of constant it is overkill

Comment (matllubos) [reply security/utils.py:349]: Especially I must here reset colors which send django. So I still will have to use RESET constant

File: security/task.py [hunk 2]
```diff
@@ -115,10 +115,8 @@ def on_apply_task(self, task_log, args, kwargs, options):
                 **(kwargs or {})
             ),
             extra=dict(
-                task_args=args,
```
Comment (matllubos) [inline security/task.py:118]: @rubickcz here

File: security/task.py [hunk 3]
```diff
@@ -115,10 +115,8 @@ def on_apply_task(self, task_log, args, kwargs, options):
...
                 task_id=task_log.celery_task_id,
                 task_input=task_log.input,
-                task_kwargs=kwargs,
```
Comment (matllubos) [inline security/task.py:121]: @rubickcz here

File: security/task.py [hunk 4]
```diff
@@ -194,9 +192,7 @@ def on_start_task(self, task_run_log, args, kwargs):
                 **(kwargs or {})
             ),
             extra=dict(
-                task_args=args,
                 task_id=task_run_log.celery_task_id,
-                task_kwargs=kwargs,
```
Comment (matllubos) [inline security/task.py:199]: @rubickcz here

File: security/task.py [hunk 5]
```diff
@@ -194,9 +192,7 @@ def on_start_task(self, task_run_log, args, kwargs):
                 **(kwargs or {})
             ),
             extra=dict(
-                task_args=args,
```
Comment (matllubos) [inline security/task.py:197]: @rubickcz here

File: security/task.py [hunk 6]
```diff
@@ -228,17 +224,14 @@ def on_success_task(self, task_run_log, args, kwargs, retval):
...
```
Comment (matllubos) [inline security/task.py:231]: @rubickcz here

File: security/task.py [hunk 7]
```diff
@@ -228,17 +224,14 @@ def on_success_task(self, task_run_log, args, kwargs, retval):
...
                 task_duration=task_run_log.time,
                 task_empty_error=task_run_log.error_message is None or len(task_run_log.error_message) < 1,
                 task_empty_output=task_run_log.output is None or len(task_run_log.output) < 1,
                 task_id=task_run_log.celery_task_id,
-                task_kwargs=kwargs,
```
Comment (matllubos) [inline security/task.py:236]: @rubickcz here

File: security/task.py [hunk 8]
```diff
@@ -228,17 +224,14 @@ def on_success_task(self, task_run_log, args, kwargs, retval):
...
                 task_log_id=None if not task_log else task_log.pk,
                 task_name=task_run_log.name,
                 task_queue=None if not task_log else task_log.queue_name,
                 task_retries=task_run_log.retries,
-                task_retval=retval,
```
Comment (matllubos) [inline security/task.py:241]: @rubickcz here

**Review by rubickcz — APPROVED** [review]

**Review by Formulka — APPROVED** [review]

**Pull request merged**